docs/machine-learning/resources/tasks.md

```diff
@@ -0,0 +1,46 @@
+# Machine Learning Tasks
+After understanding what you are hoping to achieve with your data, you will need to pick the right machine learning task for your situation. Below, you can find an explanation of different machine learning tasks and some common use cases. 
+
+> [!NOTE]
+> ML.NET is currently in Preview. All machine learning tasks are not currently supported. To submit a request for a certain task, please open an issue on our [GitHub repository](https://github.com/dotnet/machinelearning/issues).
+
+## Binary Classification
+A supervised machine learning task that is used to predict which of two classes an instance of data belongs to. The input of a classification algorithm is a set of labeled examples. Each example is represented as a feature vector, and each label is an integer of value of 0 or 1. The output of a binary classification algorithm is a classifier, which can be used to predict the label of new unlabeled instances. Examples of binary classification scenarios include:
```

Comment:
A supervised machine learning task that is -> A supervised machine learning task is a task that is (same feedback for the other introductions)

Reply:
You're right. Never mind @aditidugar.